### PR TITLE
Landing page and categories and trending quizes 

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello everyone done with the Quiz Game's Landing Page and also the categories of quiz.
Thanks a lot.
Please review this PR.

### CODE REVIEW

File: css-files/landing-page.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* Top Navigation Bar */
+.component-navigation {
+    background-color: #C9C3E6;
+    height: 13rem;
+    width: 100%;
+    box-shadow: 0 2px 3px -1px rgb(0 0 0 / 10%);
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    /* flex-wrap: wrap; */
```
Comment (ShekharDhangar) [inline css-files/landing-page.css]: Create flex as helper class and do the changes you want like space-between in the classes you want to apply. 
Also remove commented code

File: css-files/root.css [hunk 2]
```diff
@@ -71,4 +76,110 @@ a{
...
+    font-size: 2rem;
+}
+.text-p{
+    font-size: 1.6rem;
+}
+
+
+
+
+
```
Comment (ShekharDhangar) [inline css-files/root.css:93]: remove spacing

**Review by ShekharDhangar — APPROVED** [review]
Do the changes, rest is good

File: css-files/root.css (hunk 2 above)
Comment (Dev-AkshatGupta) [reply css-files/root.css:93]: I will remove it once I m done with this project . Thanks for reviewing the PR

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good. Nice work!

Your undraw image in `main`  is completely on the left. Add some padding to it

**Comment by Dev-AkshatGupta** [comment]
> Looks good. Nice work!
> 
> Your undraw image in `main` is completely on the left. Add some padding to it

Thanks a lot Apurva for reviewing my PR and I did What you told me for. Thanks

File: css-files/landing-page.css (hunk 1 above)
Comment (Dev-AkshatGupta) [reply css-files/landing-page.css]: Added a new class as you asked for it Thanks for the suggestion :)

**Pull request merged**